manticore/platforms/evm.py

```diff
@@ -816,6 +879,13 @@ def _push_results(self, instruction, result):
             assert instruction.pushes == 0
             assert result is None
 
+    def _calculate_extra_gas(self, *arguments):
+        current = self.instruction
+        implementation = getattr(self, f"{current.semantics}_gas", None)
```

Comment:
What are the possible values here? Maybe an `if, elif, else` instead?

Reply:
Ok right, I get it now. This probably slows down the execution a lot (we do some more calls and calls are  kinda expensive in Python) but let's not focus on that now.

Reply:
Hmm yes. Better ways for distpatching gas calculation are wellcome. Lets discuss it in a diff PR/Issue.

Though, we can disable it all by not calling `_calculate_extra_gas` at all when at that specific mode.

It wont be.. well.. real EVM but ppl do not care much about it.